packages/client/_src/Resources/Blog/GetPosts.ts

```diff
@@ -0,0 +1,43 @@
+import { BlogPost } from "@effect-ts-app/boilerplate-types/Blog"
+
+@allowAnonymous
+@allowRoles("user")
+export class GetPostsRequest extends Get("/blog/posts")<GetPostsRequest>()(
+  {}
+) {}
+
+export class GetPostsResponse extends ModelEnc<GetPostsResponse, GetPostsResponse>()({ // TODO MNModel<GetPostsResponse, GetPostsResponse.ConstructorInput, GetPostsResponse.Encoded, GetPostsResponse.Props> ()({
+  items: prop(array(BlogPost))
+}) {}
+/** @ignore @internal @deprecated */
+export type GetPostsResponseConstructor = typeof GetPostsResponse
```

Comment:
Why does `GetPost` have the codegenerated code when others don't?

What is the purpose of exporting the `GetPostsResponse` interface and `GetPostsResponse` namespace?

Reply:
> Why does GetPost have the codegenerated code when others don't?

It's optional, but it helps with terseness of types generated in .d.ts, and it helps with performance when using these types.
And in the past it helped with compatibility to WebStorm.

> What is the purpose of exporting the GetPostsResponse interface and GetPostsResponse namespace?

To become available on the `BlogRsc`, which is used by
- the API Controllers to implement the handling of the request and the API Router to mount at the specified path and method.
- and by the UI to get a `clientFor(BlogRsc)`